src/event-handler/on.ts

```diff
@@ -1,8 +1,11 @@
-module.exports = receiverOn;
+import { webhookNames } from "../generated/webhook-names.js";
+import { EventState } from "../types";
 
-const webhookNames = require("../webhook-names.json");
-
-function receiverOn(state, webhookNameOrNames, handler) {
+export function receiverOn(
+  state: EventState,
+  webhookNameOrNames: string | string[],
```

Comment:
shouldn't this be a generic type as done in the type definition file?

Reply:
> shouldn't this be a generic type as done in the type definition file?

You are totally right